blockbot/templates/admin.tpl

```diff
@@ -0,0 +1,4 @@
+{{include file="field_checkbox.tpl" field=$good_crawlers}}
+{{include file="field_checkbox.tpl" field=$block_gab}}
+{{include file="field_checkbox.tpl" field=$training}}
+<div class="submit"><input type="submit" name="page_site" value="{{$submit}}" /></div>
```

Comment:
Please write your base templates using [Bootstrap 3.4 syntax](https://getbootstrap.com/docs/3.4/css/#forms). It degrades nicely when the Bootstrap CSS isn't present and doesn't require to add a frio-specific template to look nice.

In this case, it just means this:
```suggestion
<div class="submit"><button type="submit" class="btn btn-primary" name="page_site" value="{{$submit}}">{{$submit}}</button></div>
```

Reply:
I just used copy&paste :-)